deploy/helm/events-collector-config.yaml

```diff
@@ -8,7 +8,7 @@ exporters:
 extensions:
   health_check:
     check_collector_pipeline:
-      enabled: true
+      enabled: false
       interval: "5m"
       exporter_failure_threshold: 5
```

Comment:
We can remove this whole section and revert it to the `health_check: {}` as it was originally.

Reply:
Removed as suggested.